src/main/java/nl/b3p/tailormap/api/persistence/services/FeatureSource.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence.services;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Basic;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OrderColumn;
+
+public class FeatureSource {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Basic(optional = false)
+    private String protocol;
+
+    @Basic(optional = false)
+    private String title;
+
+    @Basic(optional = false)
+    private String url;
+
+    @Column(columnDefinition = "jsonb")
+    private Object authentication;
+
+    @ManyToOne
+    @JoinColumn(name = "linked_service")
+    private GeoService linkedService;
+
+    @ManyToMany(cascade = CascadeType.ALL) // Actually @OneToMany, workaround for HHH-1268
```

Comment:
since https://github.com/hibernate/hibernate-orm/pull/2224 / HHH-1268 long time ago we should be able to do it right instead of porting over old workarounds

Reply:
Yeah didn't get around to that yet, I did mention it in the design document

Reply:
Noted, will do later